library/core/src/lib.rs

```diff
@@ -207,9 +198,9 @@
 #![feature(const_mut_refs)]
 #![feature(const_precise_live_drops)]
 #![feature(const_refs_to_cell)]
+#![feature(const_trait_impl)]
```

Comment:
why is this still needed?

Reply:
edit: ah nevermind, Destruct is still const_trait, but not used anywhere like that in libcore